app/scripts/components/common/blocks/scrollytelling/index.tsx

```diff
@@ -450,7 +468,11 @@ function Scrollytelling(props) {
             className='root'
             mapRef={mapRef}
             containerRef={mapContainer}
-            onLoad={() => setMapLoaded(true)}
+            onLoad={() => {
+              setMapLoaded(true);
+              // Fit the map to the container once  loaded.
+              mapRef.current?.resize();
```

Comment:
I noticed the scrollymap doesn't fill up the screen height often (as in the screenshot). This bug seems to exist in the production too,  but somehow the change on this PR makes this bug much more obvious. I think it was because the map was not properly resized when the map was loaded - so I put this line.

![Screen Shot 2023-06-06 at 12 12 30 PM](https://github.com/NASA-IMPACT/veda-ui/assets/4583806/0b4dd7b5-cf50-4174-9a2f-66be32e221e0)

Reply:
This was a great catch!